webextension/manifest.json

```diff
@@ -17,6 +17,7 @@
 	"bookmarks",
 	"history",
 	"storage",
+	"tabs",
```

Comment:
add permission `tabs` to use the property url: "<all_urls> in browser.tabs.query